### PR TITLE
[CDAP-12511] change confirmation message when a pipeline is configured to trigger …

### PR DESCRIPTION
**Pull request by elfenheart**
…other pipelines

JIRA: https://issues.cask.co/browse/CDAP-12511

This is UI change for: https://github.com/caskdata/cdap/pull/9595

### Approach
Had to refactor the Outbound Trigger component to store the triggeredPipelines list in the store. The array length is needed to check whether there is outbound triggers configured on a given pipeline or not

### CODE REVIEW

File: cdap-ui/app/cdap/components/TriggeredPipelines/index.js [hunk 1]
```diff
@@ -154,7 +128,31 @@ export default class TriggeredPipelines extends Component {
...
+        namespace={namespace}
+      />
+    </Provider>
+  );
+}
+
 TriggeredPipelines.propTypes = {
   pipelineName: PropTypes.string.isRequired,
-  namespace: PropTypes.string.isRequired
+  namespace: PropTypes.string.isRequired,
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/TriggeredPipelines/index.js]: nit: unnecessary change

File: cdap-ui/app/cdap/components/TriggeredPipelines/index.js [hunk 2]
```diff
@@ -154,7 +128,31 @@ export default class TriggeredPipelines extends Component {
   }
 }
 
+TriggeredPipelinesView.propTypes = {
+  pipelineName: PropTypes.string.isRequired,
+  namespace: PropTypes.string.isRequired,
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/TriggeredPipelines/index.js]: Are we actually using this anywhere as a prop? I only see one mention of `namespace` variable being used in this component, and we are getting that from the NamespaceStore.

**Review by tonybach — COMMENTED** [review]
Minor comments

**Review by tonybach — APPROVED** [review]
LGTM 👍

File: cdap-ui/app/hydrator/controllers/detail/top-panel-ctrl.js [hunk 3]
```diff
@@ -283,6 +283,21 @@ class HydratorDetailTopPanelController {
     delete exportConfig.__ui__;
     this.myPipelineExportModalService.show(exportConfig, exportConfig);
   }
+  getDeleteMessage() {
+    let triggeredPipelines = window.CaskCommon.TriggeredPipelineStore.getState().triggered.triggeredPipelines;
+    let count = triggeredPipelines.length;
+
+    if (count > 0) {
```
Comment (ajainarayanan) [inline cdap-ui/app/hydrator/controllers/detail/top-panel-ctrl.js:290]: Might be worth to make the final backend call to check for a list of pipelines current one will trigger. If the user deletes Pipeline1 within a UI session when in parallel someone added it as trigger can be caught. Just to stay upto-date with backend before deleting.

**Review by ajainarayanan — COMMENTED** [review]
Just one minor comment. Otherwise 👍 LGTM

**Comment by elfenheart** [comment]
On-hold, waiting for backend PR to be merged

**Comment by bdmogal** [comment]
LGTM
